content/docs/04-clusters/01-public-cloud/01-aws.md

```diff
@@ -1018,19 +1018,37 @@ sigs.k8s.io/cluster-api-provider-aws/cluster/[ClusterName] = owned
 ## Running Clusters through Edit Node Pool
 <InfoBox>
 Palette allows its users to apply/edit the Taints, for a running cluster, through the <b>Edit node pool</b> option under the <b>Nodes</b> tab.
-</InfoBox>
+</InfoBox>
+
+**Master Pool**
 
 |**Parameter**     | **Description**|
 |------------------|---------------|
 |**Name**          |A descriptive name for the node pool.|
 |**Size**          |Number of VMs to be provisioned for the node pool. For the master pool, this number can be 1, 3, or 5.|
-|**Allow worker capability (master pool)**|Select this option for allowing workloads to be provisioned on master nodes.|
+|**Allow worker capability (master pool)**|Select this option for allowing workloads to be provisioned on master nodes.|
```

Comment:
```suggestion
|**Allow worker capability **|Select this option for allowing workloads to be provisioned on master nodes.|
```
We are already in the context of Master Pool, so we don't need to specify master pool.

Why do we need to